### ISSUE TITLE
[Bug]: Accessibility. Non typical work of VM settings with screenreader

### ISSUE THREAD

**Issue by sevapopov2** [VirtualBox/virtualbox#295]
### Version

trunk/main

### Host OS Type

Windows

### Host OS name + version

Windows 11 Pro Version 24H2 (OS Build 26100.6899)

### Host Architecture

x86

### Guest OS Type

other

### Guest Architecture

x86

### Guest OS name + version

Not applicable

### Component

GUI

### What happened?

The navigation in VM settings with screenreader is not standard. It requires a lot of time to navigate between different options.

### How can we reproduce this?

Preconditions:
Screenreader is running.
There is at least 1 virtual machine created.

Steps to reproduce:

1. Open VM's settings by pressing CTRL+s.
2. Press up and down arrows in categories list.
3. Navigate in the settings window with Tab key.

Actual behavior:
Screenreader doesn't announce categories list elements when navigating with up and down arrow keys. Tab key moves between all available settings in all categories which slows down the process of navigation to the needed option.

Expected behavior:
List elements are announced by screenreader when navigating in categories list. Navigating with Tab key moves between options of the selected category and not by all options available in all categories.

Reproduced on:
NVDA 2025.3.
VirtualBox Version 7.2.3 r170871.

### Did you upload *all* of your necessary log files, screenshots, etc.?

- [x] Yes, I've uploaded all pertinent files to this issue.
